app/src/gui/components/record/view.tsx

```diff
@@ -356,28 +356,31 @@ export function ViewComponent(props: ViewProps) {
                 Other sections have errors. Click to navigate:
               </Typography>
 
-              {otherSectionsWithErrors.map((section, _index) => (
```

Comment:
Does this belong in here?

Reply:
GitHub is being silly - I was fixing all those lint warnings! It' just I added the comment and it thinks I deleted everything :)